python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -599,6 +614,66 @@ def func(x):
 
         return self.agg(func)
 
+    def describe(self):
+        """
+        Generate descriptive statistics that summarizes the central tendency,
+        dispersion and shape of a dataset’s distribution, excluding NaN values.
+
+        Analyzes numeric DataFrames only
+
+        Parameters
+        ----------
+        DataFrame to be summarized
```

Comment:
Pandas supports these parameters:
```
percentiles: list-like of numbers, optional
include: ‘all’, list-like of dtypes or None (default), optional
exclude: list-like of dtypes or None (default), optional
datetime_is_numeric: bool, default False
```
We should have the ability to support at least `percentiles` `include` and `exclude`? For `datetime_is_numeric` I need to check if cudf has chronic specific aggregation implemented.